src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -7617,9 +7617,7 @@ public Encoding Encoding
 
             set
             {
-#pragma warning disable MSLIB0001
-                if (value == System.Text.Encoding.UTF7)
-#pragma warning restore MSLIB0001
+                if (value != null && value.CodePage == 65000)
```

Comment:
I guess `65000` code page stands for UTF7, right? Can you please add a comment?

Reply:
Will do. Also see https://github.com/dotnet/runtime/issues/41459#issuecomment-682106551